geniza/corpus/models.py

```diff
@@ -173,6 +173,9 @@ class Fragment(TrackChangesModel):
 
     objects = FragmentManager()
 
+    # NOTE: may want to add optional ForeignKey to djiffy Manifest here
+    # (or property to find by URI if not an actual FK)
```

Comment:
I like the FK idea here because it links the data more tightly. Isn't it already possible to find by URI? (`iiif_url="uri"`)?

Reply:
right — convenience property would be easy and might be good enough, but if we're going to cache them our db it would probably be better to make the connection explicit